components/cli/cmd/cellery/apply_autoscale_policy.go

```diff
@@ -23,12 +23,10 @@ import (
 	"log"
 	"regexp"
 
-	"github.com/cellery-io/sdk/components/cli/pkg/kubectl"
-
-	"github.com/cellery-io/sdk/components/cli/pkg/commands"
-
 	"github.com/spf13/cobra"
 
+	"github.com/cellery-io/sdk/components/cli/kubernetes"
```

Comment:
cli/kubernetes or cli/pkg/kubernetes

Reply:
cli/kubernetes

Reply:
We have to move this to cli/pkg/kubernetes as the pkg directory is used for reusable packages